### DISCUSSION TITLE
Feature Requests - Cosmetic

### DISCUSSION THREAD

**Discussion by mkammes** [djdd87/SynoAI#87] [Ideas]
SynoAI version number in log output upon startup.

Line thickness options for Exclusion and Boundary boxes.

Thanks!

**Comment by djdd87** [comment]
StrokeWidth already exists for the line thicknesses. Although I note it's missing from the documentation.

**Comment by djdd87** [comment]
Logged #88 and #89 .

**Reply by mkammes** [reply]
Brilliant!  Thank you.  Perhaps the version number of SynoAI in the logs, too? Thanks again for the stellar work on this project. I'm enjoying it immensly.

**Reply by djdd87** [reply]
Yeah that's #89 as mentioned above. No problem. Glad it's working for you.
